cli/cmd/portfolio/cmd/seed/flag/generate.go

```diff
@@ -9,14 +9,25 @@ import (
 // Flag names.
 const (
 	mnemonicFlag = "mnemonic"
+	seedFlag = "seed"
 )
 
 // AddMnemonicFlag adds the mnemonic flag to the command
 func AddMnemonicFlag(c *cobra.Command) {
-	cliflag.AddPersistentBoolFlag(c, mnemonicFlag, false, "Description here...", false)
+	cliflag.AddPersistentBoolFlag(c, mnemonicFlag, false, "Generate mnemonic phrase", false)
+}
+
+// AddSeedFlag adds the seed flag to the command
+func AddSeedFlag(c *cobra.Command) {
```

Comment:
Recommendation to use getters and setters function together in the order. I.e. Set... then Get... the other setters and getters.